apps/demos/Demos/Splitter/Overview/ReactJs/PaneContent.js

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+
+export default function PaneContent(paneName) {
+  const dimensionOptions = new Set(['size', 'minSize', 'maxSize']);
+  return function template(data) {
+    const getPaneState = useMemo(() => {
+      if (data.resizable !== false && !data.collapsible) {
+        return 'Resizable only';
+      }
+      return `${data.resizable ? 'Resizable' : 'Non-resizable'} and ${
+        data.collapsible ? 'collapsible' : 'non-collapsible'
+      }`;
+    }, [data.resizable, data.collapsible]);
+    const filterDimensionOptions = useMemo(
+      () =>
+        Object.entries(data)
+          .filter(([key]) => dimensionOptions.has(key))
+          .map(([key, value]) => ({ key, value })),
+      [data],
+    );
```

Comment:
please, apply the suggestions which a left for text
and also add empty lines - now it looks unreadable